docs/how-to-translate-files.md

```diff
@@ -74,13 +77,7 @@ When you attempt to save a translation, you may see a warning message appear wit
 This message appears when Crowdin's QA system has identified a potential error in the proposed translation. In this example, we have modified the text of a `<code>` tag and Crowdin has caught that.
 
 > [!WARNING]
-> You have the option to save a translation in spite of errors, but this should be done only in circumstances where the error might be incorrect.
-
-Crowdin will sometimes convert HTML tags (such as `<code>`) into numerical values, like `<0>text</0>` or `<1>text</1>`. These tags are usually code tags, and our QA check will verify that these have not been translated. However, sometimes these may be `<strong>` or `<em>` tags, which _should_ be translated. If you hover over a numerical tag in the source string, you can see which HTML tag or tags have been replaced:
-
-![Image - Hover over a tag](./images/crowdin/tag-hover.png)
-
-If the replaced tags are something that should be translated (such as `strong` tags), translate the content within the tags and select "Save Anyway" when the error appears. Then leave a comment on the string explaining that the tags are `strong` (for example) so other translators and proofreaders are aware that they should be translated.
+> You have the option to save a translation in spite of errors, but this should be done only in circumstances where the error is incorrect. If you are clicking "Save Anyway", you should also tag a proofreader or project manager in a comment on that string for further review. Include the reason why you have saved the translation with QA errors.
```

Comment:
```suggestion
> You have the option to save a translation in spite of errors. If you do, by clicking "Save Anyway", you should also tag a proofreader or project manager and explain why the QA message needs to be ignored in this case.
```
I felt it was self-explanatory that you'd only want to do this if you disagreed with the QA message. Also, I've shifted the tone from "justify yourself" to "tell us what's going wrong" since we want to encourage feedback rather than make people feel defensive.

Reply:
I thought it was self-explanatory too, but the number of translations I've seen come in this week that have QA issues suggest otherwise. 😅 

I do like this verbiage tho.

Reply:
I just prefer to assume people are acting in good faith if possible.  I suspect that the people that don't much care probably aren't taking the time to read the docs, so we'd probably not reach them either way.